oxidation/parsec_cli/parsec_sequestre_dump/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "parsec_sequestre_dump"
+version = "0.0.0"
+edition = "2021"
+license = " BUSL-1.1"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+parsec_api_crypto = { path = "../../crates/parsec_api_crypto" }
+parsec_api_types = { path = "../../crates/parsec_api_types" }
+
+diesel = { version = "2.0.0-rc.0", features = ["sqlite", "chrono"] }
+clap = { version = "3.1.18", features = ["derive"] }
+serde = { version = "1.0.127", features = ["derive"] }
```

Comment:
why serde/serde_json ? deserialization should all be handled by `parsec_api_types`